js/lib/db.js

```diff
@@ -1,80 +1,27 @@
-export const data = [
-  {
-    id: 0,
-    question: 'What type of a language is HTML?',
-    answer: 'Markup Language',
-    isBookmarked: true,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 1,
-    question: 'Inside which HTML element do we put the JavaScript?',
-    answer: '<script>',
-    isBookmarked: true,
-    tags: ['HTML', 'Javascript', 'general'],
-  },
-  {
-    id: 2,
-    question: 'Who maintains the CSS specifications?',
-    answer: 'World Wide Web Consortium.',
-    isBookmarked: true,
-    tags: ['CSS'],
-  },
-  {
-    id: 3,
-    question: 'Who designed the Linux OS?',
-    answer: 'Linus Torvalds',
-    isBookmarked: true,
-    tags: ['Linux'],
-  },
-  {
-    id: 4,
-    question:
-      'Which of the following key combinations allows to terminate the current process in Linux shell?',
-    answer: 'Ctrl + C',
-    isBookmarked: false,
-    tags: ['Linux', 'Shell', 'Shortcuts'],
-  },
-  {
-    id: 5,
-    question: 'What is the correct way to write a JavaScript array?',
-    answer: 'const colors = ["red", "green", "blue"]',
-    isBookmarked: true,
-    tags: ['Javascript', 'Array', 'general'],
-  },
-  {
-    id: 6,
-    question: 'What should be the first tag in any HTML Document?',
-    answer: '<!doctype html>',
-    isBookmarked: false,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 7,
-    question: 'How can you add a comment in a JavaScript?',
-    answer: '//This is a comment',
-    isBookmarked: false,
-    tags: ['Javascript', 'general'],
-  },
-  {
-    id: 8,
-    question: 'What tag is used to display a picture in a HTML page?',
-    answer: 'Img',
-    isBookmarked: false,
-    tags: ['HTML', 'HTML-Tag', 'general', 'picture'],
-  },
-  {
-    id: 9,
-    question: 'HTML are web pages read and rendered by?',
-    answer: 'Web Browser',
-    isBookmarked: false,
-    tags: ['HTML', 'general'],
-  },
-  {
-    id: 10,
-    question: 'Who created Git?',
-    answer: 'Linus Torvalds',
-    isBookmarked: true,
-    tags: ['Git'],
-  },
-]
+export function pushSection(sectionObject) {
+  // Get the sections from the localStorage and convert them to JavaScript array
+  let sections
+
+  const storageSections = JSON.parse(localStorage.getItem('sections'))
+  if (storageSections === null) {
+    sections = []
+  } else {
+    sections = storageSections
+  }
+
+  // Add the new section to the array
+  sections.push(sectionObject)
+  // Store the array again inside the local storage
+  localStorage.setItem('sections', JSON.stringify(sections))
+}
+
+export function getSections() {
```

Comment:
Similar here, i'd call the function `getQuestions` and adapt the variable names inside the function as well